src/components/Users/UserResetPassword.tsx

```diff
@@ -192,16 +192,16 @@ export default function UserResetPassword({
 
   const editButton = () => (
     <div className="mb-4 flex justify-start">
-      <ButtonV2
+      <Button
+        variant="outline_primary"
         onClick={() => setIsEditing(!isEditing)}
         type="button"
         id="change-edit-password-button"
-        className="flex items-center gap-2 rounded-sm border border-gray-100 bg-white px-3 py-1.5 text-sm text-[#009D48] shadow-sm hover:bg-gray-50"
-        shadow={false}
+        className="flex items-center gap-2 px-3 py-1.5  shadow-sm "
```

Comment:
I don't recommend applying such classes to the button. Especially do not override shadow's and all. Let's stick with what the common designs configured by variant provides.

As for the spacing between the icon, use `ml-2` in the icon's classname instead

```suggestion
```